rpcserver.go

```diff
@@ -2790,6 +2790,12 @@ func (r *rpcServer) arbitratorPopulateForceCloseResp(chanPoint *wire.OutPoint,
 		case contractcourt.ReportOutputIncomingHtlc,
 			contractcourt.ReportOutputOutgoingHtlc:
 
+			// Don't report details on htlcs that are no longer in
```

Comment:
Is an accompanying extension to the existing integration tests planned?

Reply:
Do you mean specifically for this commit, or reporting on anchors in general? For anchors in general, my plan is to build that on top of #4001. Or possibly change a few of the asserted values in that pr when this pr lands first.

Reply:
SGTM to build on top of #4001. Timing wise, it may not need to land, but it's important that we're at least able to _run_ them all to find any potentially issues.